packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@coderscamp/ui/components/Button';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { HStack } from '@coderscamp/ui/components/Stack';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { DesktopBaseNavbar, MobileBaseNavbar } from './BaseNavbar';
+
+export const Navbar = () => {
+  const baseNavbar = useBreakpointValue({ base: <MobileBaseNavbar />, lg: <DesktopBaseNavbar /> } as const);
+  const buttonSize = useBreakpointValue({ base: 'xs', sm: 'md' } as const);
+
+  return (
+    <HStack padding="18px 2.4vw" width="100%" justifyContent="space-between" alignItems="center" position="relative">
```

Comment:
💭 `18px 2.4vw` still seems a little tricky. It would be great to use pixels of the vertical padding.

Reply:
If i use pixels for the vertical padding it will be too big for mobile or too small on desktop. I can use UseBreakpointValue() but i wouldn't say that additional logic will be better than simple vw

Reply:
Pixel values for spacing can be defined on a design system level and then unified across the entire app. When you use magic values like 2.4vw it makes space sizing unpredictable and less maintainable (imagine many components with such type of dynamic padding contained in one layout).

Of course, I agree that Chakra's way of making breakpoints creates an additional logic layer in the code. However, it's still easier to test layouts that may potentially beak only on five predefined breakpoints than layouts with many dynamic-sized elements that can break on any size of the screen.

Reply:
Just to clarify. I'm not saying that dynamic sizing is always bad. In cases where you want to divide the layout into many parts and for example use `width: 25vw` it's totally ok. In my opinion, the problem emerges when you create some dynamic values only because a particular, very precise number seems to work in a certain context.

Reply:
I understand, still not convinced that using Chakra's hook is better in this case, but i changed to pixels